modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -172,6 +172,12 @@ image::dashboard-built-in-infrastructure.png[Example: Infrastructure Dashboard]
 | *Custom metrics* a|
 
 The *Custom metrics* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. See xref:anypoint-custom-metrics-connector.adoc[Anypoint Custom Metrics Connector].
+
+| *Application Network* a|
+
+The *Application Network* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. The *Application Network* chart shows the xref:visualizer::troubleshooting-visualization.adoc[Troubleshooting visualization] for the selected app. Additionally, it shows only direct callers and callees for the selected app, and data retention is set to 7 days. You can click the *Metrics* menu to see different metrics related to the app you selected. For more information about metrics in Anypoint Visualizer, see xref:visualizer::troubleshooting-visualization.adoc#choosing-your-metric[Choosing your Metric].
```

Comment:
Your text is just fine as-is. 
Because I am a lazy reader, I like to break things up; the choice is yours.
In the following suggestion, please replace the text within the < > with something commonly useful:
```suggestion
The *Application Network* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. 

The *Application Network* chart on this tab shows the xref:visualizer::troubleshooting-visualization.adoc[Troubleshooting visualization] for the selected app. Additionally, it shows only direct callers and callees for the selected app over the most recent seven days. 

You can click *Metrics* to see statistics such <this, that, and the other thing> related to the app you selected. For more information about metrics in Anypoint Visualizer, see xref:visualizer::troubleshooting-visualization.adoc#choosing-your-metric[Choosing Your Metric].
```